### PR TITLE
Revert "Revert "Enterprise 1.9.7 release notes""

### PR DESCRIPTION
**Pull request by kelseiv**
Reverts influxdata/docs-v2#4017

### CODE REVIEW

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 1]
```diff
@@ -9,6 +9,31 @@ menu:
     parent: About the project
 ---
 
+## 1.9.7 [2022-05-26]
+
+### Features
+- Add experimental passive node feature; expose passive node feature to influxd-ctl and the API.
```
Comment (samhld) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:15]:
```suggestion
- Expose passive node feature to influxd-ctl and the API.
```

**Review by samhld — CHANGES_REQUESTED** [review]
LGTM after one change

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md (hunk 1 above)
Comment (samhld) [reply content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md]: That feature has existed for a while but it was not hidden from the CLI/API

**Review by ono760 — APPROVED** [review]
Looking good

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 2]
```diff
@@ -9,6 +9,31 @@ menu:
...
+#### Flux updates
+- Add [http requests package](/{{< latest "flux" >}}/stdlib/experimental/http/requests/).
+- Add [isType()](/{{< latest "flux" >}}/stdlib/types/istype/) function.
+- Add [display()](/{{< latest "flux" >}}/stdlib/universe/display/) function.
+- Enhancements to the following functions: [increase()](/{{< latest "flux" >}}/stdlib/universe/increase/), [sort()](/{{< latest "flux" >}}/stdlib/universe/sort/), [derivative()](/{{< latest "flux" >}}/stdlib/universe/derivative/), [union()](/{{< latest "flux" >}}/stdlib/universe/union/), [timeShift()](/{{< latest "flux" >}}/stdlib/universe/timeshift/), vectorization to applicable functions such as [map()](/{{< latest "flux" >}}/stdlib/universe/map/).
+- Add TCP connection pooling to [mqtt.publish()](/{{< latest "flux" >}}/stdlib/experimental/mqtt/publish/) function when called in a map() function.
+
+### Bug fixes
+- Fix race condition causing `influxd-ctl restore` command to fail.
+#### Error Messaging
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:27]:
```suggestion

#### Error Messaging
```

**Pull request merged**